app/src/main/java/com/example/habitsmasher/FollowUserDialog.java

```diff
@@ -145,31 +159,60 @@ public void onComplete(@NonNull Task<QuerySnapshot> task) {
     /**
      * A helper method that shows a toast message after the user follow operation is successful
      */
-    private void showFollowSuccessMessage() {
+    private void showFollowRequestSuccessMessage() {
         Toast.makeText(getContext(),
-                       USER_FOLLOWED_SUCCESS_MESSAGE,
+                      REQUEST_SENT_SUCCESS,
                        Toast.LENGTH_LONG)
              .show();
     }
 
     /**
-     * This method is responsible for adding a new user to the following array of the given user
-     * @param userId the user performing the operation
-     * @param followedUserId the followed user to add to the collection
+     * Detrimines if a user has already requested to follow this user already
+     * @param followedId user who is being sent the request
+     * @param followerId user who sent the request
+     * @return true if user already sent the request
+     */
+    private boolean searchInRequestsList(String followedId, String followerId) {
```

Comment:
Since this returns a boolean, let's rename this to `hasUserAlreadySentRequest` so it flows better.

- [ ] rename method